labman/gui/templates/sequencing.html

```diff
@@ -64,7 +64,7 @@
     $buttonElem.append("<span class='glyphicon glyphicon-remove'></span>")
     $aElem.append($buttonElem);
     $('#pool-list').append($aElem);
-    $('#addBtnPool' + poolId).prop('disabled', true);
+    // $('#addBtnPool' + poolId).prop('disabled', true);
```

Comment:
Can this comment and the one on line 76 be removed please?

Reply:
Done.